### ISSUE TITLE
[BUG] org-token missing outputs

### ISSUE THREAD

**Issue by ngortheone** [splunk-terraform/terraform-provider-signalfx#193]
Hey @cory-signalfx 
There is another one for you

I have successfully created an [org token](https://www.terraform.io/docs/providers/signalfx/r/org_token.html), but there are 2 problems:

1. [documentation page](https://www.terraform.io/docs/providers/signalfx/r/org_token.html) does not list any resource outputs.

2. When I check what is recorded in the state - there is also nothing useful there.

```
$ terraform state show module.signalfx_org_token.signalfx_org_token.token                                                                                                          

# module.signalfx_org_token.signalfx_org_token.token:
resource "signalfx_org_token" "token" {
    disabled = false
    id       = "my-cool-org-token"
    name     = "my-cool-org-token"
}

```

I need to pass the token value to the config file, but I can't even obtain it. The only way to obtain it seems to be via UI by clicking on "show token" button

![Screenshot_20200422_115115](https://user-images.githubusercontent.com/18271144/80021881-10309d80-8490-11ea-9649-ac6a672b1c1e.png)

Please add token value to the resource output so it can be used.

**Comment by cory-signalfx** [comment]
Oh dear, how embarrassing! 😸

**Comment by cory-signalfx** [comment]
Fix coming in #194

**Comment by cory-signalfx** [comment]
This has been merged and Hashi should do a release for us in the next couple of hours. I will close this, assuming they are going to get that done. Thank you for flagging @ngortheone, not sure how I missed that.
